torchvision/io/__init__.py

```diff
@@ -20,10 +23,75 @@
     encode_jpeg,
     write_jpeg,
     encode_png,
-    write_png
+    write_png,
 )
 
 
+if _HAS_VIDEO_OPT:
+
+    class Video:
+        """
+        Low level video-reading API.
+
+        Args:
+
+            path (string): Path to the video file in supported format
+
+            stream (string, optional): descriptor of the required stream. Defaults to "video:0"
+                Currently available options include :mod:`['video', 'audio', 'cc', 'sub']`
```

Comment:
For a follow-up PR: it would be good to explain a bit more the relationship with the first part (stream) and the second part (stream id)